frontend/package-lock.json

```diff
@@ -1,8 +1,16182 @@
 {
   "name": "frontend",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
I think there should not be some frontend in that  PR

Reply:
hmm, yeah, it shouldn't, I think if they did: `npm install` it changed that lock.json file...don't know why